acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/SignerConfiguration.java

```diff
@@ -52,11 +54,13 @@ public SignerConfiguration(
       final Optional<AzureKeyVaultParameters> azureKeyVaultParameters,
       final Optional<TlsOptions> serverTlsOptions,
       final Optional<TlsCertificateDefinition> overriddenCaTrustStore,
+      final String slashingProtectionDbUrl,
       final String slashingProtectionDbUsername,
       final String slashingProtectionDbPassword,
       final String mode,
       final Optional<Map<String, String>> web3SignerEnvironment,
-      final boolean enableSlashing) {
+      final boolean enableSlashing,
+      final String postfix) {
```

Comment:
What does postfix represent? This could do with a better name I think. Just not sure what it means right now.